style.css

```diff
@@ -574,8 +635,243 @@ button:active {
+  }
+
+  .top-bar-icon {
+    display: none;
+  }
+
+  .subtext {
+    font-size: 32px;
+    line-height: 44px;
+    padding-top: 148px;
+    padding-left: 143px;
+    width: 566px;
+  }
+
+  .text {
+    font-size: 20px;
+    text-shadow: 1px 1px #1c1a19;
+    margin-top: 12px;
+    padding-right: 24px;
+    padding-left: 143px;
+    font-weight: 400;
+    width: 720px;
+    display: flex;
+  }
+
+  .social-icons {
+    position: absolute;
+    top: 330px;
+    padding-left: 36px;
+    justify-content: flex-start;
+    flex-direction: column;
+  }
+
+  .background-desktop {
+    background: url("assets/background_desktop.png");
+    background-repeat: no-repeat;
+    background-color: #1c1a19;
+    background-size: 45%;
+    background-position: right bottom;
+    gap: 12px;
+  }
+
+  .line {
+    margin-top: -50px;
+    margin-left: 350px;
+  }
+
+  .text2 {
+    margin-top: -460px;
+    margin-left: 700px;
+  }
+
+  .text3 {
+    margin-left: 700px;
+  }
+
+  .link-tag {
+    margin-left: 700px;
+  }
+
+  .space {
+    padding-top: 200px;
+  }
+
+  .btn2 {
+    margin-left: 700px;
+    max-height: 48px;
+  }
+
+  .container-section1 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .container-section2 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .backgroundabout-me {
+    background: url("assets/background_desktop.svg");
+    background-repeat: no-repeat;
+    background-position: right;
+  }
+
+  .about-me,
+  .first h2 {
+    text-align: left;
+    padding-top: 130px;
+    padding-left: 24px;
+    line-height: 88px;
+  }
+
+  .about-me,
+  .second {
+    font-size: 20px;
+    line-height: 24px;
+    text-align: left;
+    text-shadow: 1px 1px #1c1a19;
+    color: #fff;
+  }
+
+  .line2 {
+    margin-top: 130px;
+    margin-bottom: 2px;
+    height: 1px;
+    background-color: #fff;
+    margin-left: 80px;
+    margin-right: 160px;
+  }
+
+  .skills {
+    display: grid;
+    padding-left: 80px;
+    padding-top: 80px;
+    grid-column-gap: 10px;
+  }
+
+  .inner-list1 {
+    margin-left: 10px;
+  }
+
+  .skilled-2 {
+    margin-left: 400px;
+    margin-top: -220px;
+  }
+
+  .inner-list2 {
+    margin-left: 410px;
+  }
+
+  .skilled-3 {
+    margin-left: 800px;
+    margin-top: -260px;
+  }
+
+  .inner-list3 {
+    margin-left: 810px;
+  }
+
+  .cnt-container {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .main-text {
+    font-size: 40px;
+    margin-top: 130px;
+    margin-left: 80px;
+    margin-right: 24px;
+    text-align: left;
+    width: 455px;
+  }
+
+  .form-container {
+    display: block;
+    margin-left: 520px;
+    margin-top: -280px;
+  }
+
+  .contact-hr {
+    width: 648px;
+  }
+
+  .form-text {
+    width: 648px;
+  }
+
+  .about-btn4 {
+    display: block;
+    margin-left: 24px;
+    border: 1px currentColor;
+  }
+
+  .hr-line-bottom {
+    width: 100%;
+  }
+
+  .social-icons-bottom {
+    display: flex;
+    margin-left: 50%;
+    margin-right: 50%;
+    width: 50%;
+  }
```

Comment:
- [ ] Let's try to fix this footer, right now is overflowing to the right of the page create blank space all over your app on the left.

![image](https://user-images.githubusercontent.com/79114233/140407292-7949bdb8-8ca0-433d-b515-a6dbe9e0b0e9.png)

Reply:
This is fixed! :+1: